src/components/create-study-form.js

```diff
@@ -273,12 +272,13 @@ export const CreateStudyForm = (props) => {
             isPrivateStudy,
             activeDirectory
         ).then((res) => {
-            setCreateStudyErr('');
-            setStudyName('');
-            setStudyDescription('');
-            dispatch(removeSelectedFile());
-
-            if (!res.ok) {
+            if (res.ok) {
+                props.setOpen(false);
```

Comment:
strange thing to edit props inside the component isn't it ?
If the 'open' state is outside this component, then we should add a `onClose` props to pass to this component and then call it nope ?

Reply:
done